src/NServiceBus.NHibernate/SharedSession/NHibernateStorageSession.cs

```diff
@@ -79,5 +77,26 @@ protected override void Setup(FeatureConfigurationContext context)
             Installer.RunInstaller = context.Settings.Get<bool>("NHibernate.Common.AutoUpdateSchema");
             Installer.configuration = configuration;
         }
+
+        static bool DisableConnectionSharing(FeatureConfigurationContext context, Configuration configuration)
+        {
+            return context.Settings.GetOrDefault<bool>("NServiceBus.Features.SqlServerTransportFeature")
+                   && context.Settings.GetOrDefault<bool>(typeof(Outbox).FullName)
+                   && (SqlServerDriver(configuration) || SqlServerDialect(configuration));
```

Comment:
I've raised a general concern regarding how we should address that https://github.com/Particular/Requirements/issues/258